### PR TITLE
fix(examples): use testURL and module names

### PR DESCRIPTION
**Pull request by posva**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Other: updated example

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] All new and existing tests are passing.

---

I updated the jest example by removing unnecessary <rootDir> that I have found to create problem when having multiple version of babel. I also added the testURL property (https://github.com/jsdom/jsdom/issues/2304)

### CODE REVIEW

**Review by TheAlexLichter — DISMISSED** [review]

**Review by pi0 — APPROVED** [review]
Thank you @posva :+1: :100:

File: examples/jest-vtu-example/package.json [hunk 1]
```diff
@@ -31,16 +31,17 @@
...
-      "<rootDir>/node_modules/jest-serializer-vue"
+      "jest-serializer-vue"
     ],
     "collectCoverage": true,
     "collectCoverageFrom": [
       "<rootDir>/components/**/*.vue",
       "<rootDir>/pages/*.vue"
-    ]
+    ],
+    "testURL": "http://localhost/"
```
Comment (pi0) [inline examples/jest-vtu-example/package.json]: Shouldn't  we specify default listen port here? (http://localhost:3000)

Comment (posva) [reply examples/jest-vtu-example/package.json]: apparently not, having just the hostname is enough to remove the `SecurityError: localStorage is not available for opaque origins` error

Comment (pi0) [reply examples/jest-vtu-example/package.json]: Thanks for the clarification.

**Pull request merged**